### PR TITLE
EOS-20617 : Disable LoggingMsgHandler, LoggingProcessor.

### PR DESCRIPTION
**Pull request by sumedhak27**
## Problem Statement:
Remove LoggingMsgHandler, LoggingProcessor imports and inclusion in config.

## Solution Design:
Code removal

## Coding:

* [x] Did you follow coding standard and verified it with flake8? 

* [x] Did your address all Codacy issues?

## Testing:

* [ ] Unittest updated and all unittests are passing?

* [x] Sanity/Self tests are updated (if applicable)?

* [ ] Manual testing done covering happy path and non-happy path?

## Integration:

* [ ] If there is any interface change, did you communicate it to other components?
* [ ] If there is any interface change, other componenent gate gatekeepers are ready to accept the change?
* [ ] Did you complete deployment test if applicable?

## PR checklist:
* [x] Did you add Jira number to the PR?

* [x] DCO and cla-signed?

### CODE REVIEW

**Review by sandeepsgt — APPROVED** [review]
Approved, Have one question though. Why we are commenting code instead of removing. ?

File: low-level/framework/messaging/egress_processor.py [hunk 1]
```diff
@@ -209,17 +209,19 @@ def _transmit_msg_on_exchange(self):
...
-                log_msg = self._jsonMsg.get("message").get("IEM_routing").get(
-                    "log_msg")
-                if self._iem_route_addr != "":
-                    self._producer.send([json.dumps(self._jsonMsg)])
-                else:
-                    logger.warn(
-                        "EgressProcessor, Attempted to route IEM without a valid 'iem_route_addr' set.")
-                logger.debug(
-                    "_transmit_msg_on_exchange, Successfully Sent: %s" % log_msg)
+            # elif self._jsonMsg.get("message").get("IEM_routing") is not None:
```
Comment (cdeshmukh) [inline low-level/framework/messaging/egress_processor.py:215]: actually remove old code. These references we can have from cortx-1.0 branch or git history also

File: low-level/framework/messaging/ingress_processor.py [hunk 2]
```diff
@@ -204,10 +204,13 @@ def _process_msg(self, body):
 
     def _send_to_msg_handler(self, msgType, message, uuid):
         # Hand off to appropriate actuator message handler
-        if msgType.get("logging") is not None:
-            self._write_internal_msgQ("LoggingMsgHandler", message)
 
-        elif msgType.get("thread_controller") is not None:
+        # DEPRICATED: LoggingMsgHandler is no more in use. IEM routing
```
Comment (cdeshmukh) [inline low-level/framework/messaging/ingress_processor.py:208]: remove unused code, here & else where too

File: low-level/framework/utils/conf_utils.py [hunk 3]
```diff
@@ -30,8 +30,6 @@
 DISKMSGHANDLER="DISKMSGHANDLER"
 IEMSENSOR="IEMSENSOR"
 IPMI="IPMI"
-LOGGINGMSGHANDLER="LOGGINGMSGHANDLER"
```
Comment (cdeshmukh) [inline low-level/framework/utils/conf_utils.py:33]: here it can stay

File: low-level/message_handlers/node_data_msg_handler.py [hunk 4]
```diff
@@ -681,6 +680,8 @@ def _send_ifdata_json_msg(self, sensor_type, resource_id, resource_type, state,
         self.if_sensor_data = jsonMsg
         self.os_sensor_type[sensor_type] = self.if_sensor_data
 
+        # DEPRICATED: LoggingMsgHandler is no more in use. IEM routing
```
Comment (cdeshmukh) [inline low-level/message_handlers/node_data_msg_handler.py:683]: spelling - DEPRECATED

File: sspl_test/framework/utils/conf_utils.py [hunk 5]
```diff
@@ -30,8 +30,6 @@
 DISKMSGHANDLER="DISKMSGHANDLER"
 IEMSENSOR="IEMSENSOR"
 IPMI="IPMI"
-LOGGINGMSGHANDLER="LOGGINGMSGHANDLER"
```
Comment (cdeshmukh) [inline sspl_test/framework/utils/conf_utils.py:33]: can stay still here

Comment (sumedhak27) [reply sspl_test/framework/utils/conf_utils.py]: added.

File: low-level/message_handlers/node_data_msg_handler.py (hunk 4 above)
Comment (sumedhak27) [reply low-level/message_handlers/node_data_msg_handler.py]: removed deprecated code.

File: low-level/framework/utils/conf_utils.py (hunk 3 above)
Comment (sumedhak27) [reply low-level/framework/utils/conf_utils.py]: added.

File: low-level/framework/messaging/ingress_processor.py (hunk 2 above)
Comment (sumedhak27) [reply low-level/framework/messaging/ingress_processor.py]: done. 👍

File: low-level/framework/messaging/egress_processor.py (hunk 1 above)
Comment (sumedhak27) [reply low-level/framework/messaging/egress_processor.py]: done 👍

**Review by cdeshmukh — APPROVED** [review]

**Review by thavanathan — CHANGES_REQUESTED** [review]

File: low-level/message_handlers/node_data_msg_handler.py [hunk 6]
```diff
@@ -681,15 +680,6 @@ def _send_ifdata_json_msg(self, sensor_type, resource_id, resource_type, state,
         self.if_sensor_data = jsonMsg
         self.os_sensor_type[sensor_type] = self.if_sensor_data
 
-        # Send the event to logging msg handler to send IEM message to journald
```
[the remark this answers is not in the document]
Comment (sumedhak27) [reply low-level/message_handlers/node_data_msg_handler.py:684]: already removed, @thavanathan

File: low-level/message_handlers/disk_msg_handler.py [hunk 7]
```diff
@@ -851,29 +849,6 @@ def _log_IEM(self, drive):
                 logger.info(f"DiskMsgHandler, Unknown disk status/reason: {status}/{reason}")
                 return
 
-        json_data = {"enclosure_serial_number": drive.get_drive_enclosure(),
```
[the remark this answers is not in the document]
Comment (sumedhak27) [reply low-level/message_handlers/disk_msg_handler.py:854]: yes this whole function can be removed, removing it.

**Review by thavanathan — APPROVED** [review]

File: low-level/message_handlers/disk_msg_handler.py [hunk 8]
```diff
@@ -36,8 +36,6 @@
 from json_msgs.messages.sensors.expander_reset import ExpanderResetMsg
 from json_msgs.messages.sensors.hpi_data import HPIDataMsg
 from json_msgs.messages.sensors.node_hw_data import NodeIPMIDataMsg
-# Modules that receive messages from this module
-from message_handlers.logging_msg_handler import LoggingMsgHandler
```
Comment (cdeshmukh) [inline low-level/message_handlers/disk_msg_handler.py:40]: disk_msg_handler is invoked even anywhere today. I think its already disabled already, in which case lets not change code here

Comment (sumedhak27) [reply low-level/message_handlers/disk_msg_handler.py:40]: It is used in DiskMonitor and NodeControllerMsgHandler currently, so I have removed only LoggingMsgHandler related code from this file for current ticket.
There is other deprecated code in this file, which can be removed in future ticket. like request/response handling regarding hpi, drive maanger, HDS etc.

Comment (cdeshmukh) [reply low-level/message_handlers/disk_msg_handler.py:40]: right, as discussed earlies, yes

**Pull request merged**